Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -3446,60 +3432,53 @@ RWMol *MolDataStreamToMol(std::istream *inStream, unsigned int &line,
   Conformer *conf = nullptr;
   try {
     if (ctabVersion == 2000) {
-      fileComplete = FileParserUtils::ParseV2000CTAB(inStream, line, res, conf,
-                                                     chiralityPossible, nAtoms,
-                                                     nBonds, strictParsing);
+      fileComplete = FileParserUtils::ParseV2000CTAB(
```

Comment:
Perhaps also make the molecule pass by reference.  Understandable if this is a private function though.

Reply:
Agreed that could be fixed at some point. Is that necessary as part of the PR to fix the main API?

Reply:
No, these are private so we can refactor later.